i3lock-fancy

```diff
@@ -113,8 +117,15 @@ else #black
         "--datecolor=00000000" "--layoutcolor=00000000")
 fi
 
-convert "$image" "${hue[@]}" "${effect[@]}" -font "$font" -pointsize 26 -fill "$bw" -gravity center \
-    -annotate +0+160 "$text" "$icon" -gravity center -composite "$image"
+if [ -n "$shot" ] ; then
```

Comment:
No need to duplicate the convert command, you can `unset` hue/effect

Reply:
Nope. The script explicitly dis-allows that on line 4 by setting option 'nounset'

Reply:
Then we should do something differently. I'm not accepting some change that duplicates half the script.